Project 3/Task 1/block_quantize.py

```diff
@@ -0,0 +1,126 @@
+import sys
+import math
+import numpy as np
+import cv2
+import matplotlib.pyplot as plt
+import re
+import Utility as util
+from os import listdir,path
+
+'''
+Quantizes a block using the number of bins
+Returns a formatted string for output to a file which details the information in the format:
+< FrameNumber, (X coordinate, Y coordinate) , Y Channel Value, Number Of Occurances >
+'''
+def quantizeBlock(block, bins, frameNum, x, y):
+    minValue = np.amin(block)
+    maxValue = np.amax(block)
+    samples, valuesPerBin = np.linspace(minValue, maxValue, bins, endpoint=False, retstep=True)
+    samples += 0.5*valuesPerBin
```

Comment:
In order to get the midtread quanta rather than the lower bound of each quanta, I move all samples up by half of a bin.